File: MovieTutorial/MovieTutorial.Web/Modules/MovieDB/Person/PersonColumns.cs
using Serenity.ComponentModel;
using System;
using System.ComponentModel;

namespace MovieTutorial.MovieDB.Columns;

[ColumnsScript("MovieDB.Person")]
[BasedOnRow(typeof(PersonRow), CheckNames = true)]
public class PersonColumns
{
    [EditLink, DisplayName("Db.Shared.RecordId"), AlignRight]
    public int PersonId { get; set; }
    [EditLink]
   // public string FirstName { get; set; }
    //Si explota es por esto:
   // public string LastName { get; set; }

    public string FullName { get; set; }
    public DateTime BirthDate { get; set; }
    public string BirthPlace { get; set; }
    [DisplayName("Height (Cm)"), Width(150)]
    public int Height { get; set; }

    public Gender Gender{ get; set; }
  
}

File: MovieTutorial/MovieTutorial.Web/Modules/ServerTypes/MovieDB/PersonColumns.ts
import { ColumnsBase, fieldsProxy } from "@serenity-is/corelib";
import { Column } from "@serenity-is/sleekgrid";
import { Gender } from "./Gender";
import { PersonRow } from "./PersonRow";

export interface PersonColumns {
    PersonId: Column<PersonRow>;
    FullName: Column<PersonRow>;
    BirthDate: Column<PersonRow>;
    BirthPlace: Column<PersonRow>;
    Height: Column<PersonRow>;
    Gender: Column<PersonRow>;
}

export class PersonColumns extends ColumnsBase<PersonRow> {
    static readonly columnsKey = 'MovieDB.Person';
    static readonly Fields = fieldsProxy<PersonColumns>();
}

[Gender]; // referenced types